public/ui.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
Is there a reason why we are putting this HTML file into the public folder? If not, revert this change.